src/Plugin/os2web/DataLookup/ServiceplatformenBase.php

```diff
@@ -236,13 +237,16 @@ private function init() {
    *   WSDL URL.
    */
   protected function getWsdlUrl() {
-    $wsdl = $this->configuration['wsdl'];
-    // If it is relative URL make is absolute.
-    if (substr($wsdl, 0, 4) !== "http") {
-      global $base_url, $base_path;
-      $wsdl = $base_url . $base_path . drupal_get_path('module', 'os2web_datalookup') . '/' . $wsdl;
+    $url = $this->configuration['wsdl'];
+    // Anything that's not an absolute path or url will be resolved relative to
+    // the location of the os2web_datalookup module.
+    if (!preg_match('@^([a-z]+:/)?/@', $url)) {
+      /** @var ExtensionPathResolver $extensionPathResolver */
```

Comment:
according to phpcs:
Data types in @var tags need to be fully namespaced.

Then that is done, then the use statement on line 5 can be removed.

Reply:
Fixed in https://github.com/OS2web/os2web_datalookup/pull/9/commits/e408c62148c1305a0cb70a87e886cda822c9c882.

Note that much of the other code in this project does does not (yet) adhere to the Drupal coding standards.